### DISCUSSION TITLE
Process for updating a brew formula (in this case protobuf)

### DISCUSSION THREAD

**Discussion by beeradmoore** [Homebrew/discussions#3269] [Everyday usage] [answered]
Hey, 
What's the general process for updating a brew formula? I use [protobuf](https://github.com/protocolbuffers/protobuf) in a few projects and I have found the formula listed [here](https://github.com/Homebrew/homebrew-core/blob/master/Formula/protobuf.rb) is out of date compared to the [latest versions](https://github.com/protocolbuffers/protobuf/releases).

[v3.20.1](https://github.com/protocolbuffers/protobuf/releases/tag/v3.20.1) appears to be the latest released but brew can only install up to [v3.19.4](https://formulae.brew.sh/formula/protobuf).

Is this something that needs a PR and if so is there documentation on what is required to change in the `protobuf.rb` file?

**Comment by SMillerDev** [comment, marked as answer]
https://docs.brew.sh/Updating-Software-in-Homebrew

**Reply by beeradmoore** [reply]
Thanks, not sure how I missed that. Happy to say that got me through to making #101683
